index.html

```diff
@@ -511,8 +511,14 @@ <h3>Use Cases and Requirements</h3>
 <a>Issuers</a> can issue revocable <a>verifiable credentials</a>.
           </li>
           <li>
-<a>Verifiable credentials</a> and <a>verifiable presentations</a> must be
-serializable in one or more interoperable machine-readable data formats.
+<a>Verifiable credentials</a> and <a>verifiable presentations</a> has to be
+serializable in one or more machine-readable data formats. The serialization
+and de-serialization process has to be deterministic, bi-directional, and lossless.
```

Comment:
"process has to" -> "processes have to"

Reply:
I think this is more correct in the singular as it's referring to "The process which consists of both serialization and deserialization" and not two unrelated processes.

Reply:
Hm.  How about "The process of serialization and/or de-serialization has to be"?

Reply:
A little wibbly for my tastes but it's understandable, so I'll incorporate it.